QuoteGeneration/pccs/container/README.md

```diff
@@ -0,0 +1,80 @@
+## 1. Build container image
+```
+docker build -t pccs:my_tag .
+```
+
+## 2. Generate certificates to use with PCCS
+```
+mkdir -p ~/pccs_tls
+cd ~/pccs_tls
+openssl genrsa -out private.pem 2048
+openssl req -new -key private.pem -out csr.pem
+openssl x509 -req -days 365 -in csr.pem -signkey private.pem -out file.crt
+rm -rf csr.pem
+```
+and give read access to the certificate/key in order they're to be readable inside container by user other than host files owner:
+```
+chmod 644 ~/pccs_tls/*
+```
+
+## 3. Fill up configuration file
+Create directory for storing configuration file:
+```
+mkdir -p ./config
+```
+Copy `<path_to_repo>/SGXDataCenterAttestationPrimitives/QuoteGeneration/pccs/config/default.json`
+to this directory:
+```
+cp <path_to_repo>/SGXDataCenterAttestationPrimitives/QuoteGeneration/pccs/config/default.json ./config/
+```
+Generate UserTokenHash:
+```
+echo "user_password" | sha512sum | tr -d '[:space:]-'
```

Comment:
```suggestion
echo -n "user_password" | sha512sum | tr -d '[:space:]-'
```

You don't want to include newlines into your password...

Reply:
Fixed.